lib/serverspec/type/linux_audit_system.rb

```diff
@@ -0,0 +1,33 @@
+module Serverspec::Type
+  class LinuxAuditSystem < Base
+    def initialize(name=nil)
+      @name = 'linux_audit_system'
+      @runner = Specinfra::Runner
+      @rules_content = nil
+    end
+
+    def enabled?
+      status_of('enabled') == '1'
+    end
+
+    def running?
+      pid = status_of('pid')
+      (!pid.nil? && pid.size > 0 && pid != '0')
+    end
+
+    private
+
+    def rules
+      if @rules_content.nil?
+        @rules_content = @runner.run_command('/sbin/auditctl -l').stdout
+#.split("\n").map(&:chomp)
+      end
+      @rules_content || []
+    end
+
+    def status_of(part)
+      cmd = "/sbin/auditctl -s | grep \"^#{part}\" | awk '{ print $2 }'"
```

Comment:
In my environment (the version of auditctl is v2.2), `auditctl -s` returns `AUDIT_STATUS: enabled=1 flag=1 pid=881 rate_limit=0 backlog_limit=320 lost=0 backlog=0` . So `grep "^enabled"` does not match even if audit system is enabled.